.github/workflows/bump-openshift-assisted-ui-lib-in-assisted-ui.yaml

```diff
@@ -0,0 +1,39 @@
+name: Bump openshift-assisted-ui-lib in assisted-ui
+
+on:
+  workflow_call:
+    inputs:
+      new-version:
+        type: string
+        required: true
+      assisted-ui-repo-url:
+        type: string
+        required: true
+      release-notes-url:
+        type: string
+        required: true
+
+jobs:
+  main:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out assisted-ui
+        uses: actions/checkout@v3
+        with:
+          repository: ${{ inputs.assisted-ui-repo-url }}
+          ref: master
+          path: ./assisted-ui
+
+      - name: Upgrade openshift-assisted-ui-lib to ${{ inputs.new-version }}
+        run: yarn upgrade --exact openshift-assisted-ui-lib@${{ inputs.new-version }}
+
+      - name: Send a pull request to assisted-ui
+        env:
+          MESSAGE: Updates openshift-assisted-ui-lib to ${{ inputs.new-version }}
+        uses: peter-evans/create-pull-request@v4
+        with:
+          base: master
+          commit-message: ${{ env.MESSAGE }}
+          title: ${{ env.MESSAGE }}
+          body: >
+            [Release notes](${{ inputs.release-notes-url }})
```

Comment:
very nice!
should it tag someone for greater visibility? for cases that you're on PTO it will be good to either tag someone or post a slack message or something alike

Reply:
Absolutely, I like the idea. I think I should use @openshift-assisted/ui-maintainers  
I just need to figure out how to send a Slack msg, do you have maybe a snippet for that?

Reply:
we can probably start with tagging this group in the PR body
not sure about slack notifications, I know how to do it with ``curl`` but there should be some github-action as well
if you want with ``curl/requests`` those are some examples:
https://github.com/openshift-assisted/assisted-installer-deployment/blob/cd98700a9aea4fd3b65912f887be63d907aa2243/tools/triage_status_report.py#L58
https://github.com/openshift-assisted/assisted-installer-deployment/blob/1fe72fd3b0f36d9d4ddf538a413050ebbff5fcb6/tools/ci_status_report.py#L182

we should also have the relevant secrets, ping me and I'll share privately